### ISSUE TITLE
Stream/Record on Linux results in `undefined symbol: obs_output_output_caption_text2`.

### ISSUE THREAD

**Issue by derickr** [ratwithacompiler/OBS-captions-plugin#43]
I get the following error when starting to record on the command line, and then OBS aborts:

```
info: recording_started_event
info: enabled: 1, is_streaming 0, streaming_output_enabled 1, streaming_transcripts_enabled 0, is_streaming_relevant: 0, is_recording 1, recording_output_enabled 1, recording_transcripts_enabled 0, is_recording_relevant: 1, is_preview_open 1, is_text_output_relevant 1, scene_collection_name: , source:  'Desktop Audio', equal_settings 1, do_captioning 1
info: caption_output_writer_loop recording starting
info: caption settings changed, starting captioning
info: source Desktop Audio active: 1
info: using direct source Desktop Audio active: 1
info: captioning start ok
info: [obs-websocket] Update << '{
    "rec-timecode": "00:00:00.000",
    "update-type": "RecordingStarted"
}'
info: ==== Recording Start ===============================================
info: [ffmpeg muxer: 'adv_file_output'] Writing file '/home/derick/media/movie/obs/2020-09-24_16-44-07.mkv'...
info: save_or_load_event_callback 1, 203759232
info: obs save event
obs: symbol lookup error: /home/derick/.config/obs-studio/plugins/libobs_google_caption_plugin/bin/64bit/libobs_google_caption_plugin.so: undefined symbol: obs_output_output_caption_text2
Couldn't initialize muxer
```

Looking at the OBS code, I think it's because I don't have OBS compiled with BUILD_CAPTIONS. It would be great if the plugin could warn about that, instead of just aborting OBS.

**Comment by ratwithacompiler** [comment]
Yeah sorry I know it's not great, initially when making the linux version I tried checking if it's available with dlsym() but could never get that to work reliably, always had some false positives or negatives so I dropped trying to check for it eventually.

I changed it now to where it should hopefully just gracefully fail to load the plugin just with an error but still start OBS normally and added a Readme explaining that to the release zip. Far from perfect but should make it a bit clearer at least. Unfortunately most distro packages of OBS don't have captions built by default from what I've seen.
